### PR TITLE
Justin/dashboard resources

### PR DESCRIPTION
**Pull request by justinjianto**

### CODE REVIEW

**Comment by justinjianto** [comment]
Created new resources page and created routes for it
Styled the resource page and added the resource videos/links

File: public/css/dashboard-resources.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.Resources {
```
Comment (jcking1034) [inline public/css/dashboard-resources.css:1]: this should probably be lowercase for consistency

File: public/css/dashboard-resources.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+.row a {
+	color: #e1489a;
+}
+
+.row p {
+	margin-top: 15px;
+	font-size: 14px;
+	font-style: normal;
+	font-weight: normal;
+	color: black;
```
Comment (jcking1034) [inline public/css/dashboard-resources.css]: would be ideal to consistently use hex for all colors used, if possible

File: views/dashboard-resources.hbs [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+    <!-- JQUERY -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+    <!-- BOOTSTRAP -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+    <!-- GOOGLE FONTS -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&family=Source+Code+Pro:wght@700&display=swap" rel="stylesheet">
```
Comment (jcking1034) [inline views/dashboard-resources.hbs:16]: Add a new line here

File: views/dashboard-resources.hbs [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+            <!--row 3 -->
+            <div class="row">
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/xPlN_Tk3VLQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <!--row 4 -->
+            <div class="row text-center">
+                <div class="col-md-6">
+                    <p>Arduino Foundations &nbsp
```
Comment (jcking1034) [inline views/dashboard-resources.hbs]: Is having %nbsp necessary?

File: views/dashboard-resources.hbs [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+            </div>
+        </div>
+    </div>
+</body>
+</html>
+
+
+
+
+{{!--
```
Comment (jcking1034) [inline views/dashboard-resources.hbs]: Delete this (and the extra newlines) since it doesn't seem to be used

**Review by jcking1034 — CHANGES_REQUESTED** [review]
Looks good for the most part, a few minor things may need to be addressed

File: public/css/dashboard-resources.css (hunk 2 above)
Comment (justinjianto) [reply public/css/dashboard-resources.css]: changed

File: views/dashboard-resources.hbs (hunk 3 above)
Comment (justinjianto) [reply views/dashboard-resources.hbs:16]: changed

File: views/dashboard-resources.hbs (hunk 4 above)
Comment (justinjianto) [reply views/dashboard-resources.hbs]: my workaround to adding a space between text and link

File: views/dashboard-resources.hbs (hunk 5 above)
Comment (justinjianto) [reply views/dashboard-resources.hbs]: good catch

**Review by jcking1034 — APPROVED** [review]

File: views/dashboard-resources.hbs [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+            <div class="row">
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/xPlN_Tk3VLQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <!--row 4 -->
+            <div class="row text-center">
+                <div class="col-md-6">
+                    <p>Arduino Foundations &nbsp
+                    <p>
```
Comment (robertnperalta) [inline views/dashboard-resources.hbs]: The closing tag isn't actually a closing tag here or on any of the other text links. It looks like Chrome automatically fixes it though. Is this some weird HTML hack I don't know?

File: views/dashboard-resources.hbs [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+</head>
+<body>
+
+    <div class="wrapper">
+        <!-- NAVBAR -->
+        {{> navbar }}
+        <!-- RESOURCES TITLE-->
+        <div class="container text-center resource-title">
+            <h1 class="Resources">Resources</h1>
```
Comment (robertnperalta) [inline views/dashboard-resources.hbs]: "Resources" should be "RESOURCES" to match the styling on other pages

**Review by robertnperalta — CHANGES_REQUESTED** [review]
Looks great! Just a few things:

* When the screen switches to mobile design, videos that are in the same row don't have any space between them. You can probably fix that with a media query and some `:nth-child` stuff
* The videos don't scale with a locked ratio with the screen, making them look off. [This website](https://avexdesigns.com/blog/responsive-youtube-embed) talks about making them responsive, and [this one](https://embedresponsively.com/) automates the whole process. See if you can work that into the code

Overall, it looks way better than I was expecting (based off of the design, not your abilities). Also, thank you for being the only person to request a review from someone other than me, and thank you @jcking1034 for actually doing it

File: views/dashboard-resources.hbs (hunk 7 above)
Comment (justinjianto) [reply views/dashboard-resources.hbs]: addressed

File: views/dashboard-resources.hbs (hunk 6 above)
Comment (justinjianto) [reply views/dashboard-resources.hbs]: addressed LOL

**Review by robertnperalta — APPROVED** [review]
I pushed a commit moving the `<style>` tags into the CSS since it was just a bunch of duplicated code. Other than that, looks great!

**Pull request merged**